<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>atpSettings</class>
 <widget class="QDialog" name="atpSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="buttonAccept">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>80</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>390</y>
     <width>80</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabSettings">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>751</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabDataBase">
    <attribute name="title">
     <string>DataBase</string>
    </attribute>
    <widget class="QLineEdit" name="editDbPath">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>541</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonOpenDb">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttoneSelectDb">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>80</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>200</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Display info for (sec):</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Path:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinSeconds">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>230</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Latin Modern Mono Prop Light</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="maximum">
      <number>60</number>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_device">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>200</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>USB</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttoneDbExport">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>80</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Export DB</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabSiteDetails">
    <attribute name="title">
     <string>Site Details</string>
    </attribute>
    <widget class="QLineEdit" name="editJobRef">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>441</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editSiteName">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>441</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="editAddres">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>271</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editPostcode">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>441</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateJobStart">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>170</y>
       <width>201</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateJobFinish">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>170</y>
       <width>201</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Reference No</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Site Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Postcode:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>140</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Job start date</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>140</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Job end Date</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
